language/diem-framework/modules/DiemAccount.move

```diff
@@ -288,7 +288,8 @@ module DiemAccount {
         payee: address,
         to_deposit: Diem<Token>,
         metadata: vector<u8>,
-        metadata_signature: vector<u8>
+        metadata_signature: vector<u8>,
+        dual_attestation: bool,
```

Comment:
Just to check my understanding: the value of `dual_attestation` determines whether this deposit needs to be dual attested. And there could be two reasons why the deposit doesn't need dual attestation: (1) the payment amount is smaller than the threshold and between same VASP, or (2) the payment transaction is multi-agent.

And here we still have metadata_signature because we want to be backwards compatible and support both regular dual attestation and multi-agent?

Reply:
Right. Plus deposit does not need dual attestation because it's not a p2p payment (like mint coins).